api/src/main/java/io/minio/CopySource.java

```diff
@@ -0,0 +1,135 @@
+package io.minio;
+
+import java.util.HashMap;
+import java.util.Map;
+import io.minio.errors.InvalidArgumentException;
+import io.minio.CopyConditions;
+/** CopySource creates the SourceObject
+ * used for Compose Object.
+ */
+public class CopySource {
+  private String bucketName;
+  private String objectName;
+  private Long offset;
+  private Long length;
+  private ServerSideEncryption sseSource;
+  private Map<String, String> headerMap = new HashMap<>();
+  private CopyConditions copyConditions;
+
+  public ServerSideEncryption getSseSource() {
```

Comment:
Have this function name as `sseSource()`

Reply:
done.